reference-architecture/aws-ansible/playbooks/openshift-setup.yaml

```diff
@@ -0,0 +1,34 @@
+- include: /usr/share/ansible/openshift-ansible/playbooks/byo/config.yml 
+  vars:
+    debug_level: 2
+    openshift_debug_level: "{{ debug_level }}"
+    openshift_node_debug_level: "{{ node_debug_level | default(debug_level, true) }}"
+    osm_controller_args:
+      cloud-provider:
+      - "aws"
+    osm_api_server_args:
+      cloud-provider:
+      - "aws"
+    openshift_node_kubelet_args:
+      max-pods:
+      - "100"
+      cloud-provider:
+      - "aws"
+      node-labels:
+      - "role={{ openshift_node_labels.role }}"
+    openshift_master_debug_level: "{{ master_debug_level | default(debug_level, true) }}"
+    openshift_master_access_token_max_seconds: 2419200
+    openshift_master_api_port: "{{ console_port }}"
+    openshift_master_console_port: "{{ console_port }}"
+    osm_cluster_network_cidr: 172.16.0.0/16
+    openshift_registry_selector: "role=infra"
+    openshift_router_selector: "role=infra"
+    openshift_master_cluster_method: native
+    openshift_cloudprovider_kind: aws
+    openshift_master_cluster_hostname: "internal-openshift-master.{{ public_hosted_zone }}"
+    openshift_master_cluster_public_hostname: "openshift-master.{{ public_hosted_zone }}"
+    osm_default_subdomain: "{{ wildcard_zone }}"
+    deployment_type: openshift-enterprise
+    openshift_master_identity_providers: [ {'name': 'google', 'kind': 'GoogleIdentityProvider', 'login': 'true', 'challenge': 'false', 'mapping_method': 'claim', 'client_id': '246358064255-5ic2e4b1b9ipfa7hddfkhuf8s6eq2rfj.apps.googleusercontent.com', 'client_secret': 'Za3PWZg7gQxM26HBljgBMBBF', 'hosted_domain': 'redhat.com'} ]
```

Comment:
probably want to pass some of these values in as variables rather than hard-coded in the template.